python/pyspark/mllib/classification.py

```diff
@@ -262,6 +262,8 @@ def load(cls, sc, path):
         model.setThreshold(threshold)
         return model
 
+    def __repr__(self):
+        return self._call_java("toString")
 
```

Comment:
I think Python style asks for two new lines here (try running ./dev/lint-python locally :))